packages/cli/src/commands/init.ts

```diff
@@ -201,7 +201,15 @@ const resolveOptionsWithPrompts = async (
     }
 
     if (!options.endpointSlug) {
-      resolvedOptions.endpointSlug = await promptEndpointSlug(path);
+      const resolvedPath = resolvePath(options.projectPath);
+      const packageJSONPath = pathModule.join(resolvedPath, "package.json");
+      const packageJSON = await readJSONFile(packageJSONPath);
+
+      if (packageJSON && packageJSON["trigger.dev"].endpointId) {
```

Comment:
This could throw an error if `packageJSON["trigger.dev"]`, could probably just rewrite this to 

```ts
if (packageJSON && packageJSON["trigger.dev"] && packageJSON["trigger.dev"].endpointId) {
```

Reply:
I'll push a change now.